helper.js

```diff
@@ -0,0 +1,46 @@
+const cliColors = {
+    red : '31m',
+    yellow : '33m',
```

Comment:
you can create a utils folder for helper.js file 
It shouldn't be at the same level as tezster.js